src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -126,6 +126,11 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    deleteCollectionLoader: gravityLoader(
+      (id) => `collection/${id}`,
+      { user_id: userID },
```

Comment:
**question**: I see that many of the restored API endpoints require a `user_id` query param (example [here](https://github.com/artsy/gravity/blob/aedb621f398c0ae8bbb69271ad90d6c3c3c53dcb/app/api/v1/collections_endpoint.rb#L116-L119) 🔒 ). Is there any context around that requirement? Since these are all authenticated endpoints, I would expect that `current_user` is always available for user-scoped collection queries and authorization checks. Seems a bit strange to take this in as an input arg here and pass that back to Gravity which should already have easy access to the user via authentication headers.

Reply:
Good question, and I'm not sure about the original motivation — I figured that general pattern was to allow for the flexibility to have users interact with _other_ users' collections. Maybe this was even part of the ancient collections implementation we had  several years ago?

Reply:
Maybe we can take another look in Gravity and simplify things there. This implementation should remain compatible, even if we remove the backend's reliance on this user_id query param. Non-blocking so feel free to merge this in when you're ready.

Reply:
Yeah, in today's demo we noticed a few other things in those old endpoints that warrant another look, I'll get these ideas into a ticket.